srv/components/provisioner/scripts/cortx_prep.sh

```diff
@@ -0,0 +1,291 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+set -euE
+export LOG_FILE="${LOG_FILE:-/var/log/seagate/provisioner/node_prep.log}"
+mkdir -p $(dirname "${LOG_FILE}")
+
+PRVSNR_ROOT="/opt/seagate/cortx/provisioner"
+localhost_ip="127.0.0.1"
+minion_id="srvnode-0"
+user_name=
+uid=
+group=
+gid=
+repo_url=
```

Comment:
Set default value as some sample URL location

Reply:
It's intentional so as to be able check whether the --target-build <url> options is provided by user or not.